salesforce/10.11/modules/ROOT/pages/salesforce-connector-reference.adoc

```diff
@@ -3199,6 +3199,186 @@ Retrieves data from specified objects, whether or not they have been deleted.
 * SALESFORCE:NOT_FOUND
 * SALESFORCE:INVALID_INPUT
 
+[[queryResultList]]
+=== Query Result List
+`<salesforce:query-result-list>`
+
+Returns a list of result IDs for a specific batch. The Salesforce API documentation states that if the batch requires retries, there will be more than one <result> element in the output. This operation should be used with <<queryResultStreamById>> to obtain the results of the query.
```

Comment:
"The Salesforce API documentation states" text is not necessary. I suggest:
"If the batch requires retries, there will be more than one <result> element in the output."

"This operation should be used with" > "Use this operation with..."